packages/grafana-ui/src/utils/dom.ts

```diff
@@ -1,41 +1,47 @@
-// Node.closest() polyfill
-if ('Element' in window && !Element.prototype.closest) {
-  Element.prototype.closest = function(this: any, s: string) {
-    const matches = (this.document || this.ownerDocument).querySelectorAll(s);
-    let el = this;
-    let i;
-    // eslint-disable-next-line
-    do {
-      i = matches.length;
+namespace DOMUtil {
```

Comment:
Love this <3 Would love to have a guid in packages readme about creating namespaces / some reference to relevant TS docs :)

Reply:
Yeah, just saw that we get a babel-build error in Circle CI. Would be super if you can help out resolving that one tomorrow @dprokop .

Reply:
Yeah, it's because of linting rule... Seems like namespaces are not recommended way for structuring what we want there: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-namespace.md